webdriver/newsession/response.py

```diff
@@ -0,0 +1,54 @@
+# META: timeout=long
+
+from util.newsession import new_session
+
+def test_resp_sessionid(_function_session):
+    resp = new_session(_function_session, {"capabilities": {}})
+    assert isinstance(resp["sessionId"], unicode)
+    uuid.UUID(hex=resp["sessionId"])
+
+
+def test_resp_capabilites(_function_session):
+    resp = new_session(_function_session, {"capabilities": {}})
+    assert isinstance(resp["sessionId"], unicode)
+    assert isinstance(resp["capabilities"], dict)
+    assert {"browserName",
+            "browserVersion",
+            "platformName",
+            "acceptInsecureCerts",
+            "setWindowRect",
+            "timeouts",
+            "proxy",
+            "pageLoadStrategy"}.issubset(
+                set(resp["capabilities"].keys()))
+
+
+def test_resp_data(_function_session, platform_name):
+    resp = new_session(_function_session, {"capabilities": {}})
+
+    assert isinstance(resp["capabilities"]["browserName"], unicode)
+    assert isinstance(resp["capabilities"]["browserVersion"], unicode)
+    if platform_name:
+        assert resp["capabilities"]["platformName"] == platform_name
```

Comment:
In case `platform_name` is not, we should still check that `platformName` is set.